assets/images/Svg.js

```diff
@@ -55,12 +56,9 @@ const svg = {
 module.exports = function(name, props) {
   // eslint-disable-line
   if (!svg[name]) {
-    const error = "Unknown SVG: " + name;
     if (isDevelopment()) {
-      throw new Error(error);
```

Comment:
this is okay. It's fine to throw an error in development. The problem is if we try to do it in production. The old version threw an error all the time.